### PR TITLE
db load averages explained

### PR DESCRIPTION
**Pull request by DaanHoogland**

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

File: source/adminguide/tuning.rst [hunk 1]
```diff
@@ -91,6 +91,24 @@ at `MySQL Reference
 Manual <http://dev.mysql.com/doc/refman/5.5/en/innodb-buffer-pool.html>`_.
 
 
+Monitor the DataBase Load
+-------------------------
+
+The load of the DB is monitorred. By default the queries for each
```
Comment (sureshanaparti) [inline source/adminguide/tuning.rst:97]: typo

```suggestion
The load of the DB is monitored. By default the queries for each
```

File: source/adminguide/tuning.rst [hunk 2]
```diff
@@ -91,6 +91,24 @@ at `MySQL Reference
...
+default. In the UI these are visible under the DB/Usage Server page
+under the infrastructure menu.
+
+.. |dbLoadAverages.png| image:: /_static/images/dbLoadAverages.png
+   :alt: load averages as displayed in the UI
+
+The configuration variable 'database.server.stats.interval' can be set
+to change the interval, which is 60 seconds by default.
+
+The value of 'database.server.stats.retention' can be changed to tweek
```
Comment (sureshanaparti) [inline source/adminguide/tuning.rst:108]:
```suggestion
The value of 'database.server.stats.retention' can be changed to tweak
```

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Review by sureshanaparti — APPROVED** [review]

**Comment by soreana** [comment]
@DaanHoogland Where I can see these values. In my setup, I don't have the DB/Usage Server page under the infrastructure menu.

**Comment by DaanHoogland** [comment]
@soreana this is part of https://github.com/apache/cloudstack/pull/5588
It is not merged yet, we are still doing internal QA for it.

**Comment by soreana** [comment]
> @soreana this is part of [apache/cloudstack#5588](https://github.com/apache/cloudstack/pull/5588)
> It is not merged yet, we are still doing internal QA for it.

@DaanHoogland Thanks a lot. I will keep an eye on that.

File: source/adminguide/tuning.rst [hunk 3]
```diff
@@ -91,6 +91,26 @@ at `MySQL Reference
 Manual <http://dev.mysql.com/doc/refman/5.5/en/innodb-buffer-pool.html>`_.
 
 
+Monitor the DataBase Load
```
Comment (nvazquez) [inline source/adminguide/tuning.rst:94]:
```suggestion
Monitor the Database Load
```

File: source/adminguide/tuning.rst [hunk 4]
```diff
@@ -91,6 +91,26 @@ at `MySQL Reference
...
+-------------------------
+
+The load of the DB is monitored. By default the queries for each
```
Comment (nvazquez) [inline source/adminguide/tuning.rst:97]:
```suggestion
The load of the database is monitored. By default the queries for each
```

**Comment by nvazquez** [comment]
@blueorangutan docbuild

**Pull request merged**